api.js

```diff
@@ -0,0 +1,28 @@
+import express from 'express';
+import crawl from './routes/crawl';
+import bodyParser from 'body-parser';
+
+function depromise(route) {
```

Comment:
would be quite nice if this function checked that `route` is indeed a promise. I would do that by just seeing if `route.then` is defined and if it's a function.